apps/institute_web/lib/institute_web/endpoint.ex

```diff
@@ -1,53 +1,69 @@
 defmodule InstituteWeb.Endpoint do
   use Phoenix.Endpoint, otp_app: :institute_web
 
-  socket "/socket", InstituteWeb.UserSocket,
+  socket("/socket", InstituteWeb.UserSocket,
     websocket: true,
     longpoll: false
+  )
 
   # Serve at "/" the static files from "priv/static" directory.
   #
   # You should set gzip to true if you are running phx.digest
   # when deploying your static files in production.
-  plug Plug.Static,
+  plug(Plug.Static,
     at: "/",
     from: :institute_web,
     gzip: false,
     only: ~w(css fonts images js favicon.ico robots.txt)
+  )
 
   # Code reloading can be explicitly enabled under the
   # :code_reloader configuration of your endpoint.
   if code_reloading? do
-    socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
-    plug Phoenix.LiveReloader
-    plug Phoenix.CodeReloader
+    socket("/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket)
+    plug(Phoenix.LiveReloader)
+    plug(Phoenix.CodeReloader)
   end
 
-  plug Plug.RequestId
-  plug Plug.Logger
+  plug(Plug.RequestId)
+  plug(Plug.Logger)
 
-  plug Plug.Parsers,
+  plug(Plug.Parsers,
     parsers: [:urlencoded, :multipart, :json],
     pass: ["*/*"],
     json_decoder: Phoenix.json_library()
+  )
 
-  plug Plug.MethodOverride
-  plug Plug.Head
+  plug(Plug.MethodOverride)
+  plug(Plug.Head)
 
   # The session will be stored in the cookie and signed,
   # this means its contents can be read but not tampered with.
   # Set :encryption_salt if you would also like to encrypt it.
 
-  session_signing_salt =
-    System.get_env("SESSION_SIGNING_SALT") ||
-      raise """
-      environment variable SESSION_SIGNING_SALT is missing.
-      """
+  env_salt = System.get_env("SESSION_SIGNING_SALT")
 
-  plug Plug.Session,
+  signing_salt =
+    cond do
+      !is_nil(env_salt) ->
+        env_salt == "true"
+
+      :dev == Mix.env() || :test == Mix.env() ->
+        # "zqM3hNve"
+        raise "salt is in dev"
+
+      :prod == Mix.env() ->
+        raise """
+        environment variable SESSION_SIGNING_SALT is missing.
+        Do not hard code this value.
+        """
+    end
+
+  plug(Plug.Session,
     store: :cookie,
     key: "_institute_web_key",
-    signing_salt: session_signing_salt
+    signing_salt: signing_salt
+  )
 
-  plug InstituteWeb.Router
+  plug(InstituteWeb.Router)
```

Comment:
If you're running `mix format` from the root of repository it should pick up the Phoenix's `.formatter.exs` which would then not add parens to this because `plug/1` is listed in `locals_without_parens`

https://github.com/phoenixframework/phoenix/blob/4c454a5744c0cbce81624e718c9954ab6a76772e/.formatter.exs

Reply:
Thanks for catching that. I had some debug left in my commits. I think I got distracted with the formatting. At least it didn't help that my changes were buried in a bunch of formatting changes. I'll take my own advice and do the formatter separate from business logic changes.